server/server/Controllers/AdminController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using MailKit.Security;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using MimeKit;
+using server.DBRepositories;
+using server.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace server.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize(Roles = "Admin")]
+    public class AdminController: Controller
+    {
+
+        private UnitOfWork db;
+        public AdminController(ApplicationContext context)
+        {
+            db = new UnitOfWork(context);
+        }
+
+        static void SendEmail(string from, string to, string name, string cakeDescription, string login, string password)
```

Comment:
Would suggest to move all logic that is related email sending to separate class to incapsulate it